src/wp-admin/includes/class-wp-posts-list-table.php

```diff
@@ -1549,6 +1547,10 @@ protected function handle_row_actions( $item, $column_name, $primary ) {
 			$actions = apply_filters( 'post_row_actions', $actions, $post );
 		}
 
+		if ( isset( $actions['inline hide-if-no-js'] ) ) {
+			echo get_inline_data( $post );
```

Comment:
```suggestion
			get_inline_data( $post );
```
No need for `echo` here, the function displays the output directly.